### PR TITLE
Create home and about pages

### PR DESCRIPTION
**Pull request by souad988**
Create home and about pages for mobile
Add speaker list dynamically
Add desktop responsiveness to home and about pages

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+                
+                <div class="partners_icon"><i class="fab fa-facebook-square"></i>acebook</div>
+                <div class="partners_icon"><i class="fab fa-google"></i>oogle</div>
+                <div class="partners_icon"><i class="fab fa-uber"></i>Uber</div>
+                <div class="partners_icon"><i class="fab fa-amazon"></i>mazone</div>
+                <div class="partners_icon">Oracle</div>
+                
+            </div>
+         </section>
+         <footer id="home_footer">
```
Comment (Deepakdanger) [inline index.html:141]: - [ ] **Required** In the Mobile version, there should be a footer on the `home page` same as `about page` with white background and logo on the left side.

File: main.css [hunk 2]
```diff
@@ -0,0 +1,314 @@
...
+.main_program_card .fas {
+  color: #f8f6f6;
+}
+
+.main_program_card p {
+  color: #f8f6f6;
+}
+
+.main_program_button {
+  background-color: #fb5205;
```
Comment (Deepakdanger) [inline main.css:139]: - [ ] **Required** The font color should be used as given in the requirement. Kindly change this. Please ensure also that all font colors should be the same as the requirement at other places.

File: README.md [hunk 3]
```diff
@@ -1 +1,79 @@
...
+
+## Live Demo
+
+[Live Demo Link](https://souad988.github.io/S-EL-DevSchool/)
+
+## Intro video
+
+> By watching this short video, you will have a better understanding of our project and mission.
+
+[S_EL_WEBDEV?]
```
Comment (Deepakdanger) [inline README.md]: - [ ] **Required** There should be a video link in `Readme` or `PR` presenting the features of the project.
- The maximum length of the video is 5 minutes.
- The video contains a demo of the project, showing the pages in different screen sizes.
- The video contains a highlight about a piece of the code the student finds interesting or is proud of making.

**Review by Deepakdanger — CHANGES_REQUESTED** [review]
Hi @souad988, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## To Highlight
- Good Commit ✔️
- Use Semantic tags ✔️
- Good Readme ✔️ 

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: index.html (hunk 1 above)
Comment (souad988) [reply index.html:141]: thanks for reviewing and for your time ,i checked the given template so the mobile home page doesn't have a footer

**Review by Hamzaoutdoors — APPROVED** [review]
Hi @souad988,   I am **Hamza** your reviewer for this PR 😸 

## Congratulations! 🎉 Project Approved ✅

_You did a great job, I really enjoyed surfing on your fancy website 😃_ 

## To Highlight
✔️ Good job implementing the less/more Speaker feature 👏🏼 
✔️ Git/GitHub flow best practices have been followed
✔️ There are no linter errors
✔️ You have a descriptive README
✔️ Wonderful UI design and good choice of theme🥇 
✔️ Great Job building your project with **Mobile-First**

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
 - Add `LICENSE` Mit to the link in your README file, because it's not professional to keep broken links in your README file ^_^
 
_Cheers and Happy coding and Keep rocking!_ 👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**